public/sw.js

```
@@ -0,0 +1,81 @@
+// @ts-nocheck it's a JavaScript file
```

ESLint

[error] Do not use "@ts-nocheck" because it alters compilation errors.

(@typescript-eslint/ban-ts-comment)